integration/common-provision.sh

```diff
@@ -65,14 +65,6 @@ pushd /tmp
 mkdir -p opencontainer-hello_def456/rootfs
 cp /usr/local/share/go/src/github.com/square/p2/integration/hello/config.json opencontainer-hello_def456/
 
-# install docker using instructions from
-# https://docs.docker.com/install/linux/docker-ce/centos/#set-up-the-repository
-sudo yum install -y yum-utils device-mapper-persistent-data lvm2
-sudo yum-config-manager \
-    --add-repo \
-    https://download.docker.com/linux/centos/docker-ce.repo
-sudo yum install -y docker-ce
-
```

Comment:
did you mean to remove this?

Reply:
Yes, since docker will be installed at the imaging step unless this is the part that does it? @eam knows more details about.

Reply:
this is run to set up your vagrant virtual machine when you run `rake integration`, so this basically _is_ the imaging process

Reply:
sorry for the confusion, rolling back my deletion.